automatic/transifex-cli/transifex-cli.nuspec

```
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>transifex-cli</id>
+    <version>1.6.4</version>
```

```suggestion
    <version>0.0.0</version>
```

don't set the latest version in the nuspec, this will actually prevent any automatic update from happening.

---

Done, thanks!